manifests/operator-manifests/mattermost/operator.yaml

```diff
@@ -14,22 +14,22 @@ spec:
         name: mattermost-operator
     spec:
       containers:
-      - args:
-        - --enable-leader-election
-        - --metrics-addr=0.0.0.0:8383
-        command:
-        - /mattermost-operator
-        env:
-        - name: MAX_RECONCILING_INSTALLATIONS
-          value: "20"
-        - name: REQUEUE_ON_LIMIT_DELAY
-          value: 20s
-        - name: MAX_RECONCILE_CONCURRENCY
-          value: "10"
-        image: mattermost/mattermost-operator:v1.19.0-rc.2
-        imagePullPolicy: IfNotPresent
-        name: mattermost-operator
-        ports:
-        - containerPort: 8383
-          name: metrics
+        - args:
+            - --enable-leader-election
+            - --metrics-addr=0.0.0.0:8383
+          command:
+            - /mattermost-operator
+          env:
+            - name: MAX_RECONCILING_INSTALLATIONS
+              value: "20"
+            - name: REQUEUE_ON_LIMIT_DELAY
+              value: 20s
+            - name: MAX_RECONCILE_CONCURRENCY
+              value: "10"
+          image: mattermost/mattermost-operator:b269a84
+          imagePullPolicy: IfNotPresent
+          name: mattermost-operator
+          ports:
+            - containerPort: 8383
+              name: metrics
```

Comment:
nit: I think these should maybe stay at their previous indentation level? Not a bit deal.

Reply:
This was made automatically from a formatter, I believe what it does is indent list items rather than having them at the same indentation level.